src/scripts/main.js

```diff
@@ -1,3 +1,168 @@
 'use strict';
+// import { pushNotification } from './notification.js';
 
-// write code here
+// Sort table
+
+const sortButtons = document.querySelectorAll('thead tr th');
+const tbody = document.querySelector('tbody');
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+
+const sortState = new Array(sortButtons.length).fill(false);
+
+sortButtons.forEach((button, index) => {
```

Comment:
Read about [event delegation](https://www.freecodecamp.org/news/event-delegation-javascript/) and try to use it here